### ISSUE TITLE
No file/schema validation on data import

### ISSUE THREAD

**Issue by joaozorro** [Capgemini/xrm-datamigration#24]
**Describe the bug**
When importing a csv file there is no validation on the schema file that os selected. you can select any type of file the tools will try to import and report no errors.

**To Reproduce**
Tab "Data Import" , Select CSV, select a txt file and continue

**Expected behavior**
File Schema validation and error 

.

**Comment by ksulikow** [comment]
@joaozorro some validation was added in the latest version, can you check if you still have this issue, please?

**Comment by ksulikow** [comment]
@tdashworth this issue is related to XrmToolbox plugin, can we move it there, please?

**Comment by tdashworth** [comment]
@ksulikow yes, this is partly for XrmToolbox but I would also say the same validation is valid for any tool (i.e. Samples or CLI).
